viewflow_extensions/management/commands/flow_graph.py

```diff
@@ -49,10 +41,15 @@ def handle(self, **options):
             raise CommandError("Could not find file %s" % (file_path, ))
         except (AttributeError, TypeError):
             raise CommandError("Could not find the flow with the name %s" % (flow_name, ))
-
-        flow_graph = FlowGraph(flow_cls)
-        graph = flow_graph.create_diagraph()
-        if options.get('svg'):
+        try:
+            from viewflow import chart
+        except ImportError:
+            from viewflow_extensions.flow_graph import FlowGraph
+            flow_graph = FlowGraph(flow_cls)
+            graph = flow_graph.create_diagraph()
             graph.render(filename='{}'.format(flow_path[0]))
         else:
-            self.stdout.write(text_type(graph))
```

Comment:
Test `test_create_graph` is failing. I think it is because in this line you removed the possibility to display graph as a string to stdout, which was the default behavior. Do we want to get rid of string representation and create always an svg? Either restore the default to print to stdout or remove the test.